### PR TITLE
fix(navbar): add link for login

### PR DESCRIPTION
**Pull request by Abecker37**
## Description

Ajout d'un lien vers la page login et change l'affichage entre profil et login en fonction de l'état de connexion 

## Type of change

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Area of change

- [x] Frontend
- [ ] Backend
- [ ] Other: .....

## General checklist:

- [x] My code follows the style guidelines of this project
- [x] I ran `bun run format`/`bun run lint` before commit
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation where needed
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [ ] I have updated `develop` and merged to my branch before submitting pull request
- [x] My pull request generate no conflicts with `develop` branch
- [x] I requested code review from other team members

## Frontend checklist:

- [x] I followed guidelines for `HTML`/`JSX`/`Typescript`
- [x] My `Javascript` generate no new console errors
- [x] I tested my code cross browsers
- [x] My slice is pixel perfect for both desktop and mobile according to design
- [x] I conducted basic QA to assure all features are working
- [x] I tested responsive for mobile and tablet resolutions

## Backend checklist:

- [ ] I tested admin by manually adding content from zero
- [ ] I created easy to use admin experience which is self-explanatory
- [ ] I added description to admin fields in hard-to-understand areas
- [ ] I followed guidelines for naming `Typescript` variables
- [ ] I conducted basic QA to assure all features are working

### CODE REVIEW

**Review by JorisPineau — APPROVED** [review]

**Review by herve-perel — DISMISSED** [review]

File: packages/frontend/web/src/components/Navbar.tsx
Comment (YDrogen) [inline packages/frontend/web/src/components/Navbar.tsx:1]: Pourquoi 2 fois `{isLoggedIn ? `
Pourquoi ne pas tout mettre dans le même ?

**Review by YDrogen — CHANGES_REQUESTED** [review]

File: packages/frontend/web/src/components/Navbar.tsx
Comment (Abecker37) [reply packages/frontend/web/src/components/Navbar.tsx:1]: Car il y a le premier qui englobe le profil coter mobile et l'autre pour le desktop

Comment (Abecker37) [reply packages/frontend/web/src/components/Navbar.tsx:1]: Après si tu à une méthode pour éviter le dédoublement de code pour le mobile et desktop je suis preneur mais je n'avais pas trouver

Comment (Abecker37) [reply packages/frontend/web/src/components/Navbar.tsx:1]: tous simplement car sur un profil j'ai une div avec le dropdown pour le desktop et un autre qui mène a la page profil pour le mobile donc je ne vois pas comment je pourrais faire ça dans une seule ternaire sachant que je veux mon dropdown que si je suis en sm md ect

**Review by JorisPineau — DISMISSED** [review]

**Review by herve-perel — APPROVED** [review]

**Review by YDrogen — DISMISSED** [review]

**Review by herve-perel — APPROVED** [review]

**Review by JorisPineau — APPROVED** [review]

**Review by YDrogen — APPROVED** [review]

**Pull request merged**